### ISSUE TITLE
nerfed gold from fish crate designs

### ISSUE THREAD

**Issue by tsunpot** [SoliaRdi/sell-angler-tokens#17]
withe patch 83.07 fish crate designs are now 10s instead of 1g making the angler tokens a fraction of their value. is it possible to change the module to grab red angleworms instead? They are 3 angler tokens per unit (instead of 10 tokens per fish crate design unit) and will yield more gold overall. And they can stack up to 300.

**Comment by SoliaRdi** [comment]
Probably. Let me look myself when patch on ru hits today

**Comment by Zezo12345** [comment]
Yeah that would be better and faster to sell in general. Inb4 they make it 10 bronze instead of silver.

**Comment by SoliaRdi** [comment]
266 gold for 8000 tokens, better to close this module =_=
